daemon/resize.go

```diff
@@ -48,6 +51,10 @@ func (daemon *Daemon) ContainerExecResize(name string, height, width int) error
 
 	select {
 	case <-ec.Started:
+		// An error may have occurred, so ec.Process may be nil.
+		if ec.Process == nil {
+			return errdefs.InvalidParameter(fmt.Errorf("exec process is not started"))
```

Comment:
can save one extra import here;

```suggestion
			return errdefs.InvalidParameter(errors.New("exec process is not started"))
```

(not a blocker, but would we want a pkg/errors error here to have a `WithStack()`) ?